src/js/yAxis.js

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Mix-in grade for value-based y axis
+    // should be used alongside the floe.chartAuthoring.axis
+    // grade in charts that require a value-based y axis
+
+    fluid.defaults("floe.chartAuthoring.yAxis", {
```

Comment:
Please reference the created JIRA https://issues.fluidproject.org/browse/FLOE-473 in a comment here